examples/functions/grouping/Netex_GroupingExample_General_01.xml

```diff
@@ -75,5 +76,21 @@ Version History
 				</GeneralGroupOfEntities>
 			</groupsOfEntities>
 		</ResourceFrame>
+		<SiteFrame version="any" id="mybus:ExampleGroupOfTariffZones">
+			<groupsOfTariffZones>
+				<GroupOfTariffZones version="any" id="mybus:InnerOslo">
+					<members>
+						<TariffZoneRef versionRef="EXTERNAL" ref="mybus:ZoneA"/>
+						<TariffZoneRef versionRef="EXTERNAL" ref="mybus:ZoneB"/>
+					</members>
+				</GroupOfTariffZones>
+				<GroupOfTariffZones version="any" id="mybus:GreaterOslo">
+					<members>
+						<TariffZoneRef versionRef="EXTERNAL" ref="mybus:ZoneC"/>
+						<TariffZoneRef versionRef="EXTERNAL" ref="mybus:ZoneD"/>
+					</members>
```

Comment:
One extra suggestion: your example will now never be part of the validation because versionRef is not checked. Would it be possible to add a TariffZoneRef that can be checked?